packages/ui/.eslintrc.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  extends: ["eslint-config-small-ads"],
```

Comment:
```suggestion
  extends: ["eslint-config-small-ads"],
  rules: {
    "import/no-extraneous-dependencies": "off",
  }
```

Reply:
Here is how we can extend the rules for a package if we wanted it. This is why I am mentioning this rule should be here only and not in the eslint package.

Reply:
Also here is where we want this bit to come 
```
  overrides: [
    {
      files: ["*.ts", "*.tsx"],
      parserOptions: {
        project: ["tsconfig.json"],
      },
    },
  ],
```

And then we we can do the same for the `src/fe..`.